code.test.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const assert = require('assert');
+eval(fs.readFileSync('code.js')+'');
+dm1 = [1,1,1,1,2,1,3,5,3,2,2,1,2,3,4,4,3,2,1];
+
+function cb1(endValue){
+	if (endValue != 7){
+		console.log("ERROR: THE END VALUE IS NOT THE CORRECT VALUE OF 7")
```

Comment:
This won't actually trigger an error in the tests. They could be failing and you would never know.